src/core/actions/addons.js

```diff
@@ -0,0 +1,29 @@
+/* @flow */
+import { FETCH_ADDON } from 'core/constants';
+import type { ErrorHandlerType } from 'core/errorHandler';
+
+type FetchAddonParams = {|
+  errorHandler: ErrorHandlerType,
+  slug: string,
+|};
+
+export type FetchAddonAction = {|
+  type: string,
+  payload: {|
+    errorHandlerId: string,
+    slug: string,
+  |},
+|};
+
+export function fetchAddon({ errorHandler, slug }: FetchAddonParams): FetchAddonAction {
+  if (!errorHandler) {
```

Comment:
Flow does protect against this but, yeah, not all components opt into Flow. Until then we'll need checks like this.